contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -168,7 +168,9 @@ function Repair-AuthorizedKeyPermission
                 $userProfilePath =  $properties.ProfileImagePath
             }
             $userProfilePath = $userProfilePath.Replace("\", "\\")
-            $fullPath -match "^$userProfilePath[\\|\W|\w]+authorized_keys$"
+            if ( $properties.PSChildName -notmatch '\.bak$') {
```

Comment:
Looks like github mistook my second slash for markup - was able to fix it.  I am suggesting that the second literal slash that is found in $properties.profileimagepath be left intact so that profiles with ".something" are not picked up as they are generally not in live use.